website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -130,27 +130,30 @@ $ cdktf deploy --help
+      --log-level                               Which log level should be written. Only supported via setting the env CDKTF_LOG_LEVEL                                                                                                                                                      [string]
+  -a, --app                                     Command to use in order to execute cdktf app                                                                                                                                                                                             [required]
+  -o, --output                                  Output directory for the synthesized Terraform config                                                                                                                                                             [required] [default: "cdktf.out"]
+      --auto-approve                            Auto approve                                                                                                                                                                                                             [boolean] [default: false]
+      --outputs-file                             Path to file where stack outputs will be written as JSON                                                                                                                                                                                   [string]
+      --outputs-file-include-sensitive-outputs   Whether to include sensitive outputs in the output file                                                                                                                                                                  [boolean] [default: false]
+      --ignore-missing-stack-dependencies       Don't check if all stacks specified in the command have their dependencies included as well                                                                                                                              [boolean] [default: false]
+      --parallelism                             Number of concurrent CDKTF stacks to run. Defaults to infinity, denoted by -1                                                                                                                                                [number] [default: -1]
+      --refresh-only                            Select the "refresh only" planning mode, which checks whether remote objects still match the outcome of the most recent Terraform apply but does not propose any actions to undo any changes made outside of Terraform.  [boolean] [default: false]
+      --terraform-parallelism                   Customize number of parallel graph traversals by Terraform. By default, the parallelism flag is not forwarded to Terraform. Note: This flag is not supported by remote / cloud backend                                       [number] [default: -1]
```

Comment:
hmm is there a way that we can explain what we mean by "parallel graph traversals"? I think this might tie into my comment below a bit, but for someone who isn't super familiar with Terraform, we may want to explain a bit more in plain language.

Reply:
Yeah. I totally get what you're saying here. I mostly tried to summarise [this documentation](https://developer.hashicorp.com/terraform/internals/graph#walking-the-graph), but overall this is hard to explain.

A few options could be:
- Say something like: `Forwarded to the terraform cli's '-parallelism' flag when set, read up more on the terraform cli documentation` and not go into the graph traversals at all?
- Keep the CLI output short, and maybe as-is, but add a separate section about parallelism underneath usage in this doc?